Engine/Commands/InvokeScriptAnalyzerCommand.cs

```diff
@@ -325,11 +326,50 @@ protected override void BeginProcessing()
                                                     ? rulePaths
                                                     : rulePaths.Concat(settingsCustomRulePath).ToArray();
             }
-            catch
+            catch (Exception e)
             {
-                this.WriteWarning(String.Format(CultureInfo.CurrentCulture, Strings.SettingsNotParsable));
-                stopProcessing = true;
-                return;
+                string errorId;
+                ErrorCategory errorCategory;
+                switch (e)
```

Comment:
I don't think it is worthwhile trying to interpret/translate the exception types since the exception message should tell what is wrong and users will see the exception type anyway in the error.

Reply:
Would it be preferable to universally identify and categorize the error as follows instead, even though some common classes of common exceptions can be reasonably classified?

```powershell
var errorId = "SettingsNotLoadable";
var errorCategory = ErrorCategory.NotSpecified;
```

Reply:
Yes, something like that. Simplicity is usually better if possible. Even if you have identify 1 case then you'd still make assumptions about and the cause and some of those exceptions (e.g. InvalidOperation) are so generic that there could be cases where a different problem is causing them.